salesagility/SuiteCRM
# Hotfix#1854

## Description



The issue was that the form was not correctly assigning the event record id before making a request to update the delegate record.

## Motivation and Context

Fixes the actions in the top button of the sub panel.

## How To Test This


1. Create an event
2. Add a couple of delegates
3. Mark the first one
4. Choose "Manage Delegated"
5. Pick a status
6. Check that the field has changed in the subpanel

## Types of changes


### Final checklist
